### DISCUSSION TITLE
When to use Extras Layer

### DISCUSSION THREAD

**Discussion by gwlester** [aws-powertools/powertools-lambda-python#535] [Q&A] [answered]
When should a Lambda use the Extras Layer vs the smaller "plain" layer?

I can't find exactly what "dependencies" the Extras Layer has.

**Comment by heitorlessa** [comment, marked as answer]
Hi Gerald - thanks for flagging it. Atm, extra means Pydantic do you can
use the Parser utility.

Could you create a documentation issue for us to clarify it, please?

**Reply by gwlester** [reply]
Done: **Please document what "extras dependencies" are included in the Extra layer** #538
